doc/source/learning/airship.rst

```diff
@@ -0,0 +1,177 @@
+Airship
+=====================
```

Comment:
I think the length of these lines are meant to match the headings.

Reply:
Correct, depending on the warning level it might even break docs! :D